hall-of-fame.md

```diff
@@ -0,0 +1,34 @@
+# Vincent Schippefilt, the 10 Feb 2023 at 10:52 AM
+
+## (fin 2020)
+
+puisqu'on a des nouveaux, je met le "wall-of-fame according to VSC":
+
+- Pierre (pro) : 1er membre de l'équipe, à implémenté la plus grosse partie de =PIVOT, et les systèmes des registries tout entier
+- Lucas (lul) : à implémenté la dernière version du composer, des graph, le input selection et les 😠 de templates avec FLE
+- Florent (FLE) : à implémenté la plupart des manipulations de grilles et tout ce qui en découle dans le différents plugins, et les 🥲 de templates avec LUL
+- Alexis (LAA) : à implémenté toutes les formules (+/- 160 à ce jour)
+- Géry (GED) : à implémenter tout. GED est un dieu vivant et nous pauvre mortels ne pouvons qu'espérer arriver un jour à sa cheville. Maintenant il est retourné Guruté l'équipe framework JS que j'ai abandonné lâchement en créant l'équipe BI avec PRO
+- Vincent (VSC): à implémenté la première version du composer, des graphs et du conditional formatting. Je suis aussi fier d'avoir trouvé l'algo de remplissage des pivot (implémenté par PRO) et le système de plugins (implémenté par GED), ainsi que les quelques refactorings qu'on a prévu de faire.
+- PRO: global filter, autofill pivots
```

Comment:
[this](https://en.wikipedia.org/wiki/Dissociative_identity_disorder) ?

source: I know people in the psychiatry field